### DISCUSSION TITLE
不可以加载多个pb文件吗？

### DISCUSSION THREAD

**Discussion by freedom12** [starwing/lua-protobuf#173] [General]
“二进制流中是什么样的schema，就会载入什么样的schema。通常只能载入一个文件。”
文档里这句只能载入一个文件，是什么意思？
如果我连续pb.load(pb.io.read("XXX.pb")了多个pb文件，会有什么问题吗？

**Comment by starwing** [comment]
是**一次调用**只能载入一个pb文件啦

**Comment by freedom12** [comment]
感谢~
